docs/gaia/gaia.rst

```diff
@@ -17,19 +17,22 @@ how to acknowledge and cite Gaia data.
 .. _guide: https://gea.esac.esa.int/archive/documentation/credits.html
 
 This package allows the access to the European Space Agency Gaia Archive
-(https//gea.esac.esa.int/archive/).
+(https://gea.esac.esa.int/archive/).
 
 Gaia Archive access is based on a TAP+ REST_ service. TAP+ is an extension of
-Table Access Protocol (TAP: https//www.ivoa.net/documents/TAP/) specified by the
+Table Access Protocol (TAP_) specified by the
 International Virtual Observatory Alliance (IVOA_).
 
-.. _IVOA: https//www.ivoa.net
+.. _TAP: https://www.ivoa.net/documents/TAP/
```

Comment:
There are multiple TAP references (even in this same filee, but there maybe one already in the other docs files) so adding these breaks the docs build. Please either reuse the exising reference or use one-off links as before (albeit without the typo of course)